From 6c8e0c22ec3a6d46aaddc3d9d190202446843c26 Mon Sep 17 00:00:00 2001
From: Hauke Mehrtens <hauke.mehrtens@intel.com>
Date: Tue, 26 Feb 2019 16:57:19 +0100
Subject: [PATCH] br_forward: Add condition around dest declaration

This fixes a compile warning when CONFIG_MCAST_HELPER is not defined.
---
 net/bridge/br_forward.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/net/bridge/br_forward.c b/net/bridge/br_forward.c
index 53e2b78d7324..7a4781b23b89 100644
--- a/net/bridge/br_forward.c
+++ b/net/bridge/br_forward.c
@@ -129,7 +129,9 @@ static int deliver_clone(const struct net_bridge_port *prev,
 			 struct sk_buff *skb, bool local_orig)
 {
 	struct net_device *dev = BR_INPUT_SKB_CB(skb)->brdev;
+#ifdef CONFIG_MCAST_HELPER
 	const unsigned char *dest = eth_hdr(skb)->h_dest;
+#endif
 
 	skb = skb_clone(skb, GFP_ATOMIC);
 	if (!skb) {
